specification/devcenter/DevCenter/DevBox/models.tsp

```diff
@@ -299,15 +394,33 @@ is detected.
   gracePeriodMinutes?: int32;
 }
 
+@doc("The Schedule list result.")
+model ScheduleListResult is Azure.Core.Page<Schedule>;
```

Comment:
Does this need an `@added` decorator?

Reply:
No, we have list schedules for both 2023 and 2024 apis. We just re-wrote the 2023 to not use stand-op now that we need to re-use the scheduleList for 2024